client/go/cmd/cert.go

```diff
@@ -38,34 +62,45 @@ environment variables. This can be useful in continuous integration systems.
 
 Note that when overriding key pair through environment variables, that key pair
 will always be used for all applications. It's not possible to specify an
-application-specific key.`
-
-func init() {
-	certCmd.Flags().BoolVarP(&overwriteCertificate, "force", "f", false, "Force overwrite of existing certificate and private key")
-	certCmd.MarkPersistentFlagRequired(applicationFlag)
-	deprecatedCertCmd.Flags().BoolVarP(&overwriteCertificate, "force", "f", false, "Force overwrite of existing certificate and private key")
-	deprecatedCertCmd.MarkPersistentFlagRequired(applicationFlag)
-}
-
-func certExample() string {
-	return "$ vespa auth cert -a my-tenant.my-app.my-instance"
-}
-
-var certCmd = &cobra.Command{
-	Use:               "cert",
-	Short:             "Create a new private key and self-signed certificate for Vespa Cloud deployment",
-	Long:              longDoc,
-	Example:           certExample(),
+application-specific key.`,
+	Example: `$ vespa auth cert -a my-tenant.my-app.my-instance
+$ vespa auth cert -a my-tenant.my-app.my-instance path/to/application/package`,
 	DisableAutoGenTag: true,
 	SilenceUsage:      true,
 	Args:              cobra.MaximumNArgs(1),
 	RunE:              doCert,
 }
 
 var deprecatedCertCmd = &cobra.Command{
-	Use:               "cert",
-	Short:             "Create a new private key and self-signed certificate for Vespa Cloud deployment",
-	Long:              longDoc,
```

Comment:
Why duplicate the documentation?

Reply:
The documentation for `auth cert` is updated with the `--no-add` argument, but I avoided adding the new functionality to the deprecated command.